com.calibre_ebook.calibre.yaml

```diff
@@ -30,13 +30,17 @@ modules:
       - tar -xvf calibre*.txz -C "${FLATPAK_DEST}/lib/calibre"
       - rm -f calibre*.txz
 
+      # Reuse liberation font files from runtime (saves 4MB)
+      - rm "${FLATPAK_DEST}/lib/calibre/resources/fonts/liberation"/*.ttf
+      - ln -s /usr/share/fonts/liberation-fonts/*.ttf "${FLATPAK_DEST}/lib/calibre/resources/fonts/liberation/"
+
       # Ensure that Calibre install script and the `xdg-utils` it uses will not
       # report errors because they are unable to find any suitable destination
       # directory to place their files into
       - mkdir -p "${FLATPAK_DEST}/bin" "${FLATPAK_DEST}"/share/{applications,desktop-directories,icons/hicolor,mime/packages}
 
       # Run Calibre system integration scripts
-      - /app/lib/calibre/calibre_postinstall --root="${FLATPAK_DEST}"
+      - '"${FLATPAK_DEST}/lib/calibre/calibre_postinstall" --root="${FLATPAK_DEST}"'
```

Comment:
This change is unrelated to the PR.

Reply:
That's right although it's small cleanup change that makes manifest consistent and may not deserve separate PR which then will trigger another update for all users.

What concerns me a little is why it needs single quotation (`'`)while all other command in manifest don't?

Reply:
Because YAML otherwise gets confused about the double quote and throws a syntax error when it “unexpectedly” stops before the end of the line. (Did /me mention they are not a huge fan of YAML already? :wink:)

> That's right although it's small cleanup change that makes manifest consistent and may not deserve separate PR which then will trigger another update for all users.

Technically, if we merge 2 PRs in less than 3 hours (cool-down or whatever it was called), it will only be pushed as one update by Flathub. Practically speaking, I agree that this shouldn't need its own PR. Maybe if I split it into a separate commit instead?

Reply:
Does yaml needs double quotes and brackets at all (like shell script)? Currently manifest look quite chaotic when sometimes we have `"${FLATPAK_DEST}"` and sometimes `$FLATPAK_DEST`.